### PR TITLE
Developer event cycle

### PR DESCRIPTION
**Pull request by AdamIsrael**
Minor cleanup

### CODE REVIEW

**Comment by chuckbutler** [comment]
:+1:  LGTM

**Comment by evilnick** [comment]
:strawberry:

File: src/en/developer-event-cycle.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Reference section of the Juju documentation.
 
 When deploying a charm, there is a guaranteed set lifecycle events that every
 charm will run - this is the basic series of hooks executed in the initial
-deployment cycle.  
+deployment cycle.
```
Comment (mbruzek) [inline src/en/developer-event-cycle.md]: Actually in this case the two spaces at the end of a line have a meaning to start a new paragraph.  I have had some problems with numbered lists right after text paragraphs so I use the two spaces on purpose as I did here.

File: src/en/developer-event-cycle.md [hunk 2]
```diff
@@ -61,16 +61,16 @@ When a relation is added from the vanilla charm to a database charm, the first
 event is `database-relation-joined` the two units know about each other and the
 code should prepare communication between the two. After the join a
 `database-relation-changed` state is set, the two units have peer to peer
-communication.  
+communication.
```
Comment (mbruzek) [inline src/en/developer-event-cycle.md]: Again used the trailing spaces on purpose to create a new paragraph.

**Comment by mbruzek** [comment]
Hey Adam thanks for fixing this stuff I use the trailing double space on purpose here to create new paragraphs when doing the editing.  Some markdown converters are more strict on this than others.

**Pull request merged**